keyboards/wilba_tech/wt_rgb_backlight.c

```diff
@@ -701,6 +705,135 @@ const is31_led __flash g_is31_leds[DRIVER_LED_TOTAL] = {
     { 1, C9_16, C7_15, C6_15 },
     { 1, C8_16, C7_16, C6_16 }
 };
+
+#elif defined(RGB_BACKLIGHT_PORTICO75)
```

Comment:
@wilba have you approved this change?

Reply:
I approve this change, but why is this being allowed and not @yiancar 's PCBs, e.g. #13130 ???

Reply:
I don't want to speak out of turn, but what I was told and the same explanation is listed in the PR you linked:

"At the very least, RGB Matrix support should also be added to the board, so that either implementation can be supported, with this block moved to the VIA's rules.mk file instead."

Now I did attack the "either implementation" a little differently than drashna suggested - the way I did it makes it an easy no to yes toggle to switch over completely from your wt_rgb_backlight.c implementation to QMK's RGB matrix implementation clear through to ifdef'ed keymaps with the appropriate keycodes.

I would assume that yiancar's PR would have been approved if he had structured his PR as requested.

Reply:
So the directive of @skullydazed from #10777 that no one other than myself can add PCBs that use the WT RGB code is now changed? I thought QMK was forcing deprecation, but now it's OK to use it as long as the default is the Core RGB code?

Reply:
There was spirited discussion around this topic when I implemented on the TKC Portico at 
https://github.com/qmk/qmk_firmware/pull/13241

Reply:
I don't think that there is a good solution nor answer here.  

The best way to support per key RGB with VIA is via the wilba_tech rgb code.  However, as the code isn't part of the core code, having more and more boards link to it isn't maintainable, nor is having that code copied to local keyboard folders. 

Unfortunately, I did not recall skullydazed's commend in the other PR, and we should have codified that requirement, to prevent such things from popping up again. 

To that end: #15128

Reply:
@drashna so what are the implications of #15128?  No future board merges with VIA Matrix RGB support unless Wilba develops them?

Reply:
#15128 clarifies the official stance as per @skullydazed's comment in Oct '20 -- pretty sure it's a case of @zvecr being unaware of that decision.

Reply:
@TerryMathews I guess this means you should revert #13241

Reply:
> I guess this means you should revert #13241

For now, I'd say no to reverting it. It's already in, and if if we revert one board, then we should also revert: 
* hs60v2
* keebwerk/mega/ansi
* nebula12
* nebula68
* nk65
* nk87
* xelus/dawn60/rev1

But that also leaves those boards without RGB.  

If you want to put in the work, and make sure they all work with the core rgb matrix feature, then I'd be happy to merge the PRs. 

But I don't think it's fair to you or us to put that work on either party here. 

Also, at this point, most of the wt_ files (main, backlight, etc) are less feature rich versions of core features.  And in #10777: 

> QMK's position regarding cross-keyboard code sharing has been consistent from the start- we don't allow it, except on a case by case basis to facilitate development while a feature is prepared for core. We gave a rare exception to the VIA developers, with the expectation that eventually that code would move to core.

> It has been more than a year and no progress has been made towards that goal. There isn't even a plan to do so as far as we know. Accordingly, we are pushing back harder and harder because this situation is not tenable in the long term. It hinders development of core for fear of breaking non-core code. What we're looking for to move forward here is a commitment to move the boards currently referencing the willba_tech vender folder over to the core RGB matrix feature.

This was posted over a year ago, by skullydazed, as well. There has been no solution yet. There has been no discussion of a direction.  And no apparent work to this end.  And to be blunt, if that code had been removed, and a better solution implemented, this wouldn't be an issue, in the first place. 

But I understand that life is busy and messy, and many times, things don't go according to plan.   

But do you have an eta or even an outline for this?


That said, @TerryMathews, it would be appreciated if you could remove the wt code.

Reply:
> #15128 clarifies the official stance as per @skullydazed's comment in Oct '20 -- pretty sure it's a case of @zvecr being unaware of that decision.

Honestly, that's more of my fault, not zvecr's. I missed or forgot what skully had said. So that's on me.

Reply:
I was aware of the previous discussions. However sometimes boards are submitted by other users, some sort of magic inner circle, and didnt want to rule that case out. My initial comment `have you approved` was more flagging it as a potential issue, vs `do you approve` that its maybe been interpreted as.

Reply:
perhaps having both versions is a solution? My `xelus/dawn60/rev1` is the WT_RGB implementation and `xelus/dawn60/rev1_qmk` is the QMK core implementation of RGB matrix.